.github/workflows/test-e2e-junk.yaml

```diff
@@ -0,0 +1,75 @@
+name: TestSuite E2E Junk
+
+on:
+  pull_request:
+    branches:
+      - "main"
+  workflow_dispatch:
+    inputs:
+      k8s_version:
+        description: With Kubernetes version
+        required: false
+
+jobs:
+  wait-for-image-build:
+    name: Wait for image build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: kyma-project/lifecycle-manager/.github/actions/wait-for-image-build@main
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          statusName: pull-runtime-watcher-img-build
+
+  e2e-integration:
+    name: E2E
+    needs: wait-for-image-build
+    strategy:
+      matrix:
+        e2e-test:
+          - watcher-enqueue
+          - watcher-metrics
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout runtime-watcher
+        uses: actions/checkout@v4
+
+      - name: Checkout lifecycle-manager
+        uses: actions/checkout@v4
+        with:
+          repository: kyma-project/lifecycle-manager
+          path: lifecycle-manager
+
+      - name: Get configuration
+        uses: kyma-project/lifecycle-manager/.github/actions/get-configuration@main
+        id: configuration
+  
+      - name: Setup tools
+        uses: kyma-project/lifecycle-manager/.github/actions/setup-tools@main
+        with:
+          k8s_version: ${{ steps.configuration.outputs.k8s_version }}
+          istio_version: ${{ steps.configuration.outputs.istio_version }}
+          k3d_version: ${{ steps.configuration.outputs.k3d_version }}
+          go-version-file: runtime-watcher/go.mod
+          cache-dependency-path: runtime-watcher/go.sum
+
+      - name: Setup test clusters
+        uses: kyma-project/lifecycle-manager/.github/actions/setup-test-clusters@main
+        with:
+          k8s_version: ${{ steps.configuration.outputs.k8s_version }}
+          cert_manager_version: ${{ steps.configuration.outputs.cert_manager_version }}
+          additional_skr_args: "-p 9090:9090@loadbalancer;-p 2112:2112@loadbalancer"
+
+      - name: Patch lifecycle-manager
+        uses: ./.github/actions/patch-lifecycle-manager
+
+      - name: Deploy lifecycle-manager
+        uses: kyma-project/lifecycle-manager/.github/actions/deploy-lifecycle-manager@main
+        with:
+          klm_version_tag: latest
+          klm_image_repo: prod
+
+      - name: Run '${{ matrix.e2e-test }}' test
+        working-directory: runtime-watcher
+        run: |
+          make -C tests/e2e ${{ matrix.e2e-test }}
```

Comment:
comment: previously we also logged pods and deployment: https://github.com/kyma-project/runtime-watcher/blob/main/.github/workflows/test-e2e-runtime-watcher.yml#L159-L162

By re-using the deploy-lifecycle-manager action, we now also wait for the pod to get up in this step (see [here](https://github.com/kyma-project/lifecycle-manager/blob/main/.github/actions/deploy-lifecycle-manager/action.yaml#L21)). Therefore I opted to not copy this logging. But happy to copy it as well if you think this is necessary